demos/vc_issuer/vc_demo_issuer.did

```diff
@@ -20,11 +20,11 @@ type Icrc21ConsentMessageResponse = variant {
 };
 type Icrc21ConsentPreferences = record { language : text };
 type Icrc21Error = variant {
-    GenericError : Icrc21ErrorInfo;
+    GenericError : record { description : text; error_code : nat };
     UnsupportedCanisterCall : Icrc21ErrorInfo;
-    ConsentMessageUnavailable : Icrc21ErrorInfo;
+    ConsentMessageUnavailable : Icrc21ErrorInfo
```

Comment:
nit: I guess the  missing trailing `;` is not obligatory, but it is usually there when automatically generating the interface.